pkgs/servers/web-apps/mealie/default.nix

```
@@ -0,0 +1,123 @@
+{ python310, python310Packages, fetchFromGitHub, mkYarnPackage, lib, ...}: let
```

Use the non-versioned `python3` please:

Also it is customary to put `lib` as the first argument, usually ASCII-betical attributes after that, and no `...` in packages:

```suggestion
{ lib, fetchFromGitHub, mkYarnPackage, python3, python3Packages }: let
```